packages/migrations/src/20210923155000-add-first-explanation-to-equation.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * This file is part of Serlo.org.
+ *
+ * Copyright (c) 2013-2021 Serlo Education e.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @copyright Copyright (c) 2013-2021 Serlo Education e.V.
+ * @license   http://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://github.com/serlo-org/serlo.org for the canonical source repository
+ */
+import { createEdtrIoMigration, updatePlugins } from './utils'
+
+createEdtrIoMigration({
+  exports,
+  migrateState: updatePlugins({
+    equations: ({ state }) => {
+      if (typeof state !== 'object' || state === null)
+        throw new Error('Illegal equation state')
+
+      return { firstExplanation: { plugin: 'text' }, ...state }
```

Comment:
This isn't good enough, is it? Don't we need to move the explanations "up"?

Reply:
Also: did you test this with equations that already have a `firstExplanation`? Code looks fine but just to make sure.

Reply:
Yeah, after a conversation with you I have add `firstExplanation` to avoid a breaking change in the plugin state (my understanding after our conversation was that then I would not need a database migration). Now we need a database migration anyway so I nothing gained from this :see_no_evil:

Reply:
Since I have worked on this issue already for a long time I would like to have it like this and to move the migration to another time...